frontend/src/metabase/components/ChannelSetupMessage/ChannelSetupMessage.jsx

```diff
@@ -27,7 +42,7 @@ export default class ChannelSetupMessage extends Component {
         <div>
           {channels.map(c => (
             <Link
-              to={"/admin/settings/" + c.toLowerCase()}
+              to={CHANNEL_MAP[c.toLowerCase()].link}
```

Comment:
Can we add validation that `c.toLowerCase()` is a valid key in `CHANNEL_MAP`? Or at the very least make sure this is wrapped in an error boundary so that if we somehow get an invalid channel name we don't break the whole page?

Reply:
Fair enough haha